config/locales/candidate_interface/degree.yml

```diff
@@ -84,6 +84,7 @@ en:
         hint: For example, 2019
       review:
         complete_hint_text: Check the entry requirements for your chosen course. Providers usually ask for a degree at 2:2 or above. Contact the training provider if you do not have the right degree level.
+        tda_complete_hint_text: Check the entry requirements for your chosen course. Postgraduate courses usually require a bachelor’s degree at 2:2 or above. Contact the training provider if you do not have the right degree level. Teacher degree apprenticeships do not require a degree.
```

Comment:
I know I'm being pedantic about this, but the name of this hint only makes sense because of the feature flag. When TDA becomes business as usual, this is just a hint and it will seem odd that it references tda. Can we change this to `hint_text`, or just `hint`? 

I am picky about naming because it is very confusing to see things in the code base now that were named for a specific set of circumstances that no longer exist. It increases the time we work on things because it makes the code difficult to read.

Reply:
I agree but the problem is that we have two hint texts. And naming  one `complete_hint_text` and another just `hint` could also confuse.

I am struggling with the name tbf. Between tda_* or undergraduate_postgraduate_* and etc.

I am assuming that when apply opens in October **we will remove the feature flag and then we will have only one hint text.**

Do you think is better maybe adding namespace like?

```
review:
  complete_hint_text: '..."
  teacher_degree_apprenticeship_feature_flag_on:
     complete_hint_text: ''
```

Then when we clean we can just remove this `teacher_degree_apprenticeship_feature_flag_on`. Wdyt?

Reply:
Done! I did what I suggested above.

Reply:
i think this is good and then you can remove it when the flag is off.